src/docs/user/references/Notification-service.adoc

```diff
@@ -495,4 +495,65 @@ Presets are not individual event types like the ones described above; instead, t
 | `All with issues`
 | Jobs
 | Automatically adds the events JOB_IN_ERROR, JOB_FINISHED_WITH_ISSUES, TASK_IN_ERROR, TASK_FINISHED_WITH_ERRORS to the subscription
-|===
\ No newline at end of file
+|===
+
+[[microsoft_teams]]
+=== Microsoft teams notifications
+
+To receive Microsoft Teams notifications using the Notification Third-Party notification method, some configuration must be applied in the Microsoft and the Notification Service sides. +
+These configurations are described in the admin guide link:../admin/ProActiveAdminGuide.html#microsoft_teams_configuration[here] and is mandatory to be able to receive Teams notifications.
```

Comment:
"These configurations are described in the admin guide (link: ../admin/ProActiveAdminGuide.html#microsoft_teams_configuration[here]) and are mandatory to receive Teams notifications."